router/router.go

```diff
@@ -0,0 +1,21 @@
+package router
+
+import (
+	"log"
+	"net/http"
+
+	"academy/controller"
+
+	"github.com/gorilla/mux"
+)
+
+// InitServer will execute in the defined port
+func InitServer() {
+
+	router := mux.NewRouter().StrictSlash(true)
+	router.HandleFunc("/api/v1/jokes", controller.JokesGetJokes).Methods("GET")
+	router.HandleFunc("/api/v1/jokes/{id}", controller.JokesGetOneJoke).Methods("GET")
+	router.HandleFunc("/api/v1/update-jokes", controller.UpdateGetData).Methods("GET")
+
+	log.Fatal(http.ListenAndServe("localhost:3000", router))
```

Comment:
Remember to start your server on the main function

Reply:
Changed to match this, thinking I might need to add some config file and load variables from there